test/sql-tap/decimal.test.lua

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/env tarantool
+local build_path = os.getenv("BUILDDIR")
+package.cpath = build_path..'/test/sql-tap/?.so;'..build_path..'/test/sql-tap/?.dylib;'..package.cpath
+
+local test = require("sqltester")
+test:plan(43)
+
+local dec = require("decimal")
+local dec1 = dec.new("111")
+local dec2 = dec.new("55555")
+local dec3 = dec.new("3333")
```

Comment:
I'd also add dec4 with 0 value. There are several places where 0 is threatened as special case.

Reply:
Added.